html-lists/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>html-list</title>
+  <body>
+    <h1><B>HTML Lists</B></h1>
+    <h2><b>Ordered List Example</b></h2>
```

Comment:
You don't need to use the `<b>` tags to make them bold, the `h1` and `h2` elements are bold by default